cli/net.go

```diff
@@ -275,6 +275,10 @@ var NetDisconnect = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
```

Comment:
nope, check how `cctx.Args()` is used below, it can be any number; it'll work fine with zero, but we probably should report misuse in that case so you could do `== 0`, and even do the `Hint` variant with `"must provide at least one peerID"`